### PR TITLE
generate custom repository when user is created [fixes SCI-2020]

### PR DESCRIPTION
**Pull request by ZmagoD**
https://biosistemika.atlassian.net/browse/SCI-2020

### CODE REVIEW

File: app/utilities/first_time_data_generator.rb [hunk 1]
```diff
@@ -25,6 +25,88 @@ def seed_demo_data(user)
...
+        last_modified_by: user,
+        repository_column: repository_column_sample_groups,
+        repository: repository
+      )
+      repository_items_sample_groups << item
+    end
+
+    # Generate random custom respository sample names and assign sample types
+    # and groups
+    repository_sample_name = (0...3).map{65.+(rand(26)).chr}.join << '/'
```
[the remark this answers is not in the document]
Comment (mlorb) [reply app/utilities/first_time_data_generator.rb]: Listen to the :dog:

File: app/utilities/first_time_data_generator.rb [hunk 2]
```diff
@@ -25,6 +25,88 @@ def seed_demo_data(user)
...
+    for i in 1..5
```
[the remark this answers is not in the document]
Comment (mlorb) [reply app/utilities/first_time_data_generator.rb]: :dog2: is the master.

File: app/utilities/first_time_data_generator.rb [hunk 3]
```diff
@@ -25,6 +25,91 @@ def seed_demo_data(user)
...
+        repository_column: repository_column_sample_groups,
+        repository: repository
+      )
+      repository_items_sample_groups << item
+    end
+
+    repository_rows_to_assign = []
+    # Generate random custom respository sample names and assign sample types
+    # and groups
+    # (0...3).map{65.+(rand(26)).chr}.join << '/'
```
Comment (mlorb) [inline app/utilities/first_time_data_generator.rb]: Remove the comment.

Comment (ZmagoD) [reply app/utilities/first_time_data_generator.rb]: probably we will do that when we will remove Samples completely

**Pull request merged**